### ISSUE TITLE
Output

### ISSUE THREAD

**Issue by ABausG** [burhankhanzada/firebase_app_distribution_github_action#6]
Hi!
Would it be possible for you to set the `firebase_console_uri`, `testing_uri`, `binary_download_uri` as outputs of the action to use them in further steps of the job?

**Comment by burhankhanzada** [comment]
yeah surely i will try to add this functionality but it may get late coz i am too busy with my university
